core/src/main/java/com/sequenceiq/cloudbreak/structuredevent/db/LegacyStructuredEventRepository.java

```diff
@@ -26,9 +26,9 @@
     @Query("SELECT se from StructuredEventEntity se WHERE se.workspace.id = :workspaceId AND se.id = :id")
     StructuredEventEntity findByWorkspaceIdAndId(@Param("workspaceId") Long workspaceId, @Param("id") Long id);
 
-    List<StructuredEventEntity> findByWorkspaceAndResourceTypeAndResourceId(Workspace workspace, String resourceType, Long resourceId);
+    List<StructuredEventEntity> findByWorkspaceAndResourceId(Workspace workspace, Long resourceId);
```

Comment:
Unfortunately, this is not going to be unique. It can return multiple resources if the if matches. Maybe it would be better to deprecate this kind of access.

Reply:
done